client/utils/challengeTypes.js

```diff
@@ -60,7 +64,7 @@ exports.submitTypes = {
   // a hosted URL where the app is running live
   // project code url like GitHub
   [backEndProject]: 'project.backEnd',
-
+  [pythonProject]: 'project.backEnd',
```

Comment:
btw, I used `project.backEnd` here on purpose because the fields submitted for Python projects are identical to back end projects.